fontforge/psread.c

```diff
@@ -2520,14 +2531,13 @@ static void _InterpretPS(IO *wrapper, EntityChar *ec, RetStack *rs) {
 	  case pt_currentmiterlimit:
 	    /* push 10.0 (default value). I don't handle this properly */
 	    if ( sp<sizeof(stack)/sizeof(stack[0]) ) {
-		stack[sp].u.val = 10.0;
+		stack[sp].u.val = (miterlimit==JLIMIT_INHERITED?10.0:miterlimit);
 		stack[sp++].type = ps_num;
 	    }
 	  break;
 	  case pt_setmiterlimit:
-	    /* pop one item off stack */
-	    if ( sp>=1 )
-		--sp;
+	    if ( sp>=0 )
+		miterlimit = stack[--sp].u.val;
```

Comment:
This doesn't look right, if sp is 0 you're addressing -1 into the stack?

Reply:
Yeah that was just a typo. Fixed.